### DISCUSSION TITLE
Small help needed

### DISCUSSION THREAD

**Discussion by waaldiigam** [adgranger/foundryvtt-dnd5e-lang-fr-fr#77] [General]
Hi there! 
I am doing polish translation for DND5e but I can't translate rules. Can you tell us how did you manage to get rules translation? How did you export it?

**Comment by adgranger** [comment]
Hello, 
Good luck for the translation ;) 
First of all, I use babele module https://gitlab.com/riccisi/foundryvtt-babele
To translate rules, I check on system repository https://github.com/foundryvtt/dnd5e/tree/3.2.x/packs/_source/rules
Each json files is a page to translate.
In the translation module, I have a file dnd5e.rule.json which must contains all rules translated (still not finished ...)
![image](https://github.com/adgranger/foundryvtt-dnd5e-lang-fr-fr/assets/131141262/079f1042-ab1f-46f4-a2ed-63e1c4fde490)

I can help you if you already have a repository created
